### ISSUE TITLE
Look into KittenPHP, KPHP, Vkontakte PHP to C++ compiler

### ISSUE THREAD

**Issue by treffynnon** [treffynnon/Benchmark-PHP-HHVM-Zephir#6]
There has been another project recently released https://github.com/vk-com/kphp-kdb

Most docs are in Russian at the moment though so could be problematic.

**Comment by treffynnon** [comment]
After trying to read this: https://raw.github.com/vk-com/kphp-kdb/master/docs/ru/README

I think the dependencies are the following for Ubuntu:

https://code.google.com/p/re2/downloads/list for regular expression parsing

``` bash
sudo apt-get install liblzma-dev libaspell-dev libreadline-dev libz-dev libfuse-dev libc6-dev
```

Still won't build. Meh.

``` bash
vv/vv-tl-parse.c: In function ‘tl_default_parse_function’:
vv/vv-tl-parse.c:219:12: error: ‘TL_ENGINE_STAT’ undeclared (first use in this function)
vv/vv-tl-parse.c:219:12: note: each undeclared identifier is reported only once for each function it appears in
vv/vv-tl-parse.c: In function ‘tl_store_stats’:
vv/vv-tl-parse.c:620:19: error: ‘TL_STAT’ undeclared (first use in this function)
make: *** [objs/vv/vv-tl-parse.o] Error 1
```

**Comment by treffynnon** [comment]
Looking for a pull request on this one as I don't have any more time for writing this benchmarking suite. @vk-com an interest?

**Comment by treffynnon** [comment]
Closing as won't fix
